### DISCUSSION TITLE
[Feature] Image Location Name/Description

### DISCUSSION THREAD

**Discussion by jamesharding26** [immich-app/immich#12289] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Because Immich doesn't use google maps, often you have to find a location like a restaurant or building by a general location nearby, or close village name. It would be great if once you have pinpointed the geographical location, you could also input a description of the location. So that in years to come, you can remember you were at 'The Oak Pub' in Towhurst, rather than just the land near it.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10739
